tests/README.md

```diff
@@ -46,11 +46,27 @@ bats test_wrapper.bats
 - Error handling (missing files, invalid paths, special characters)
 - Command construction (argument ordering, bind mount syntax, environment propagation)
 
+## Runtime Integration Tests
+
+Runtime tests verify the plugin works with actual Slurm daemons:
+
+```bash
+cd tests/runtime
+./run-tests.sh
+```
+
+These tests:
+1. Start a Slurm cluster (slurmctld + slurmd) in Docker
+2. Build and install the plugin
+3. Verify plugin CLI options in `sbatch --help`
+4. Check plugin loading in slurmd logs
+5. Submit and run a containerized test job
```

Comment:
This information is duplicated in the runtime/ README. replace this with a link to the specific README